### PR TITLE
Update python version

### PR DESCRIPTION
**Pull request by rbubley**

### CODE REVIEW

**Comment by lipis** [comment]
still red though :/

**Comment by lipis** [comment]
https://github.com/travis-ci/travis-ci/issues/10273#issuecomment-431597230

File: .travis.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ node_js:
   - '8'
 
 python:
-  - '2.7'
+  - '2.7.15'
```
Comment (lipis) [inline .travis.yml]: if this line works we won't need the sudo..

**Comment by lipis** [comment]
no luck.. :/

**Comment by rbubley** [comment]
OK - we are getting python 2.7.14 now...

**Comment by lipis** [comment]
https://github.com/travis-ci/travis-ci/issues/10273#issuecomment-431607146

**Comment by lipis** [comment]
Still failing.. :/

What if we change the language to Python?

**Comment by rbubley** [comment]
It's hanging during pip install -- let's see what the pip messages are...

**Comment by rbubley** [comment]
It seems to stall at

```
Collecting MarkupSafe>=0.23 (from jinja2==2.10->-r requirements.txt (line 7))
  Downloading https://files.pythonhosted.org/packages/4d/de/32d741db316d8fdb7680822dd37001ef7a448255de9699ab4bfcbdf4172b/MarkupSafe-1.0.tar.gz
```

I have no idea why.

**Comment by lipis** [comment]
Very strange indeed

**Comment by rbubley** [comment]
All green, @lipis !

**Pull request merged**